recipes/krbcontext/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "krbcontext" %}
+{% set version = "0.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/krbcontext-{{ version }}.tar.gz
+  sha256: f7f1bdc33a86f18f3d119a631eb250d1389378fdaea1a983eef8ef0996d64ab8
+
+build:
+  skip: True  # [win]
```

I'm pretty sure this can be noarch:
```suggestion
  noarch: python
```

We can ignore any build failures due to missing dependencies in that case.

---

@dopplershift - it turns out that it imports `pwd`, which is  for Unix password database interaction.

Given that requirement (I'm not the author of krbcontext) it would seem that there's a "don't even try using this on Windows" requirement.

The conda-forge docs seemed to suggest that adding `skip: True # [win]` to the build instructions would indicate that this is not a Windows package. Is that correct ... or should I go with noarch (and hint in some other way that this should be *nix only?